product-backend/src/main/java/org/kainos/ea/db/JobSpecificationDao.java

```diff
@@ -0,0 +1,33 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.JobSpecification;
+import org.kainos.ea.exception.RoleNotExistException;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JobSpecificationDao {
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public JobSpecification getJobSpecification(Connection c, int role_id) throws SQLException, RoleNotExistException {
+        Statement st = c.createStatement();
+        ResultSet rs = st.executeQuery("SELECT role_Id, summary, description, sharepoint_link FROM Specifications Where id =" +role_id );
```

Comment:
Update this code to be protected against SQL injection + minor rename
```suggestion
        ResultSet rs = st.executeQuery("SELECT role_id, summary, description, sharepoint_link FROM Specifications Where id =" +role_id );
```